dashboard/src/app/ide/ide.service.js

```diff
@@ -96,38 +99,48 @@ class IdeSvc {
     this.$log.error(error);
   }
 
-  startIde(noIdeLoader) {
+  startIde(workspace, noIdeLoader) {
```

Comment:
selectedWorkspace contains only the lastest workspace which is starting. I need workspace parameter to use it in chain of promises as current workspace.